config/config.exs

```diff
@@ -23,15 +23,17 @@ config :logger,
 
 config :sentry,
   dsn: {:system, "SENTRY_DSN"},
-  environment_name: Mix.env(),
   enable_source_code_context: true,
+  environment_name: {:system, "APP_ENV"},
```

Comment:
I see we use both `{:system, ...}` and `System.get_env()` in this config file. Should stick with one style? I think `{:system, ...}` loads the config at runtime and `System.get_env()` at build time?

Reply:
Yep good call. Talked with @InoMurko and moving this towards `System.get_env` until we figure out OTP releases and using a config provider.